ale_linters/php/phpcs.vim

```diff
@@ -1,15 +1,24 @@
-" Author: jwilliams108 <https://github.com/jwilliams108>
+" Author: jwilliams108 <https://github.com/jwilliams108>, Eric Stern <https://github.com/firehed>
 " Description: phpcs for PHP files
 
 let g:ale_php_phpcs_standard = get(g:, 'ale_php_phpcs_standard', '')
 
+function! ale_linters#php#phpcs#GetExecutable(buffer) abort
+    return ale#path#ResolveLocalPath(
+    \   a:buffer,
+    \   'vendor/bin/phpcs',
+    \   'phpcs'
+    \)
+endfunction
+
 function! ale_linters#php#phpcs#GetCommand(buffer) abort
     let l:standard = ale#Var(a:buffer, 'php_phpcs_standard')
     let l:standard_option = !empty(l:standard)
     \   ? '--standard=' . l:standard
     \   : ''
 
-    return 'phpcs -s --report=emacs --stdin-path=%s ' . l:standard_option
+    return ale_linters#php#phpcs#GetExecutable(a:buffer)
```

Comment:
You should `ale#Escape` the executable here, as the path might contain spaces, etc.